app/jobs/membership_event_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Documentation: https://developer.github.com/v3/activity/events/types/#membershipevent
+class MembershipEventJob < ApplicationJob
+  queue_as :github_event
+
+  def perform(payload_body)
+    return true unless payload_body.dig("action") == "removed"
+
+    user_id = payload_body.dig("member", "id")
+    team_id = payload_body.dig("team", "id")
+
+    user = User.find_by(uid: user_id)
+    return true if user.present?
```

Comment:
I think the switches between if/unless and blank/present reversed the logic of this line (and the one for group below).

Reply:
But it still works though 🤔